pitop/pma/light_sensor.py

```diff
@@ -15,6 +15,9 @@ class LightSensor(ADCBase):
     :param str port_name: The ID for the port to which this component is connected
     """
 
+    def __init__(self, port_name, pin_number=1, name="light_sensor"):
```

Comment:
Is this file missing a state property?

Reply:
nope, this will use the one in `ADCBase`